### ISSUE TITLE
Update repeatedly in error

### ISSUE THREAD

**Issue by liuxingYuuu** [DTStack/ko#109]
问题描述： 疯狂自动更新，日志如下

分支：tag/test_5.2.2

```javascript
<w> [webpack.cache.PackFileCacheStrategy] Restoring failed for Compilation/modules|/Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/antd@3.26.13_react-dom@16.13.1+react@16.13.1/node_modules/antd/es/affix/utils.js from pack: Error: Cannot find module '/Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/mini-css-extract-plugin@2.6.1_webpack@5.73.0/node_modules/mini-css-extract-plugin/dist/CssDependency'
<w> Require stack:
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/serialization/ObjectMiddleware.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/util/serialization.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/util/makeSerializable.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/WebpackError.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/HookWebpackError.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/Cache.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/Compiler.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/ProgressPlugin.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/index.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/ko@6.3.3_19b6ef28ca4d7495d716a38f13b2742e/node_modules/ko/lib/actions/dev.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/ko@6.3.3_19b6ef28ca4d7495d716a38f13b2742e/node_modules/ko/lib/cli.js
<w> (during deserialization of Array)
<w> (during deserialization of webpack/lib/NormalModule)
<w> (during deserialization of Map)
<w> (during deserialization of webpack/lib/cache/PackFileCacheStrategy PackContentItems)
<w> [webpack.cache.PackFileCacheStrategy] Restoring failed for ResolverCachePlugin|normal|default|dependencyType=|esm|path=|/Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/apps/tag/src/views/tagMarket/rowLevel|request=|@/api/apiMap from pack: Error: Cannot find module '/Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/mini-css-extract-plugin@2.6.1_webpack@5.73.0/node_modules/mini-css-extract-plugin/dist/CssDependency'
<w> Require stack:
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/serialization/ObjectMiddleware.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/util/serialization.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/util/makeSerializable.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/WebpackError.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/HookWebpackError.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/Cache.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/Compiler.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/ProgressPlugin.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/webpack@5.74.0_@swc+core@1.2.211/node_modules/webpack/lib/index.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/ko@6.3.3_19b6ef28ca4d7495d716a38f13b2742e/node_modules/ko/lib/actions/dev.js
<w> - /Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/ko@6.3.3_19b6ef28ca4d7495d716a38f13b2742e/node_modules/ko/lib/cli.js
<w> (during deserialization of Array)
<w> (during deserialization of webpack/lib/NormalModule)
<w> (during deserialization of Map)
<w> (during deserialization of webpack/lib/cache/PackFileCacheStrategy PackContentItems)
<w> [webpack.cache.PackFileCacheStrategy] Restoring failed for ResolverCachePlugin|normal|default|dependencyType=|esm|path=|/Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/apps/tag/src/views/tagMarket/rowLevel|request=|@/components/dtTable from pack: Error: Cannot find module '/Users/xuxiaoqi/Documents/work/daishu-code/dt-insight-studio/node_modules/.pnpm/mini-css-extract-plugin@2.6.1_webpack@5.73.0/node_modules/mini-css-extract-plugin/dist/CssDependency'
```

**Comment by liuxingYuuu** [comment]
I didn't know how to upload a video, so I sent it to you privately

**Comment by xbrave** [comment]
fixed in ko@6.4.0
